docs/compiling-a-contract.rst

```diff
@@ -29,7 +29,10 @@ Include the ``-f`` flag to specify which output formats to return. Use ``vyper -
 
 ::
 
-    $ vyper -f abi,bytecode,bytecode_runtime,ir,asm,source_map,method_identifiers yourFileName.vy
+    $ vyper -f abi,abi_python,bytecode,bytecode_runtime,interface,external_interface,ast,ir,ir_json,ir_runtime,hex-ir,asm,opcodes,opcodes_runtime,source_map,method_identifiers,userdoc,devdoc,combined_json,layout yourFileName.vy
+
+.. note::
+    The ``opcodes`` and ``opcodes_runtime`` output of the compiler has been returning incorrect opcodes since ``0.2.0`` due to a lack of 0 padding (patched via `PR 3735 <https://github.com/vyperlang/vyper/pull/3735>`_). If you rely on these functions for debugging, please use the latest patched versions.
```

Comment:
i don't think the output is incorrect, the formatting is just confusing.

Reply:
ok looked into #3735 a bit further, there can be ambiguous encodings sometimes which are fixed by the PR

Reply:
So I think the wording is valid, as proposed in this PR.